javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,183 @@
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = names.indexOf("Ahmad");
+
+  if(nameToRemove>-1){
+      names.splice(nameToRemove,1); //remove one item only
+  };
+  
+  console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+  //When will we be there? exircise 
+
+
+  const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+
+  function calculateTime(speed, km){
+const timeInMinutes =(km/speed)*60; // turn the num to minutes
+const hours = Math.floor(timeInMinutes/60); //gets the hours
+const rhours= Math.floor(hours);  // should give us the hours
+const minutes =Math.floor(hours-rhours)*60; //takes the decimal part and multiplate it by 60
+const rminutes= Math.floor(minutes); // should give us the minutes
+console.log (`${rhours} hours and ${rminutes} minutes`);
+return;
+    
+    };  
+
+
+calculateTime(travelInformation.destinationDistance, travelInformation.speed);
+ //not sure why the output is always 0 I tried many ways to solve it..
+  
```

Comment:
Good work Hanna, I can see you have worked on how to approach the logic.  There are several ways to go about it. I can hint you in a direction that you can try out; 
If you calculate destinationDistance/speed you get the hours in decimal numbers which would be 8.64
If you use Math.floor() on this, you would be left with only 8 hours. 
The next step would be to store the remaining 0.64 hours in a variable and convert them into minutes. 
- How could you do this? 😊

Lastly, you would use the console.log statement as you have written it. Good to see you apply template literals 👌
Let me know, if you would like to discuss this further - feel free to reach out to me on Slack, you can find me as 'Louie Meyer' 😊

Reply:
Ah thank you a lot , now it makes more sense